staging/src/k8s.io/kubectl/pkg/explain/v2/funcs.go

```diff
@@ -56,6 +56,9 @@ func WithBuiltinTemplateFuncs(tmpl *template.Template) *template.Template {
 			return "", errors.New(message)
 		},
 		"wrap": func(l int, s string) (string, error) {
+			// realign all line breaks
+			s = strings.ReplaceAll(s, "\n", "")
```

Comment:
@ah8ad3 Thanks for the quick fix! This could have been my fault when opening the issue, but the example I provided had trailing spaces, which would be unlikely in a CRD. Instead, the newline really should be replaced by a space, otherwise words from subsequent lines would get mashed together.

I'd also like to note that this is a good MVP but does not account for the various syntax rules that would require preserving _some_ newlines for Markdown (such as list items), which is technically supported in CRD descriptions.